notebooks/ima_viewer/ima_viewer.ipynb

```diff
@@ -0,0 +1,587 @@
+    "plot = make_interactive_plot(filepath, ext_type='SCI')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bd09c20e",
+   "metadata": {},
+   "source": [
+    "<a id=\"save\"></a>\n",
+    "### 2.3 Save files\n",
+    "\n",
+    "Running the cell below will generate options for saving your IMA as frames, a GIF, or both. The limits, colormap, scaling, etc. will reflect whatever is currently set above. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "b7c8fc61",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "d74839fb8783444eab64c637fae34b84",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "VBox(children=(HBox(children=(VBox(children=(HBox(children=(HTML(value='<style>p{word-wrap: break-word}</style…"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "gif_setup = gif_settings_widgets(plot)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f7d517e4",
+   "metadata": {},
+   "source": [
+    "When the IMA viewer and the widget settings are to your liking, run the following two cells to \n",
+    "1. Make the frames. \n",
+    "2. Make the gif. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "c9cda4ba",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Writing to directory: idpq32w0q_0-1023_0-1023\n",
+      "Will not overwrite any existing files.\n"
+     ]
+    }
+   ],
+   "source": [
+    "gif_frames = make_frames(plot, gif_setup)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "dc345210",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "GIF saved.\n"
+     ]
+    }
+   ],
+   "source": [
+    "make_gif(gif_frames, gif_setup)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ffdef2a4",
+   "metadata": {},
+   "source": [
+    "<a id=\"conclusions\"></a>\n",
+    "## 3. Conclusions \n",
+    "\n",
+    "Thank you for walking through this notebook! You have:\n",
+    "- learned about the format and purpose of IMA files\n",
+    "- examined different reads of an IMA file\n",
+    "- created image frames and/or GIFs for a particular customized view.  \n",
+    "\n",
+    "**Congratulations, you have completed the notebook.**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "705a5746",
+   "metadata": {},
+   "source": [
+    "<a id=\"add\"></a>\n",
+    "## Additional Resources \n",
+    "\n",
+    "Below are some additional resources that may be helpful. Please send any questions through the [HST Help Desk](https://stsci.service-now.com/hst).\n",
+    "\n",
+    "- [Types of WFC3 Files](https://hst-docs.stsci.edu/wfc3dhb/chapter-2-wfc3-data-structure/2-1-types-of-wfc3-files)\n",
```

Comment:
Putting our website, IHB, and DHB keeps all the notebooks consistent. Under the DHB link, I'd just add which section to find types of WFC3 files